app/src/main/java/org/schabi/newpipe/info_list/holder/CommentsInfoItemHolder.java

```diff
@@ -5,6 +5,8 @@
 import android.widget.ImageView;
```

Comment:
Is there a particular reason you added this code in `CommentsInfoItemHolder` and not `CommentsMiniInfoItemHolder` like all other comments code?

Reply:
It was probably left over from something else I was trying to do, should be straight forward to move it to where it is used